folium/plugins/__init__.py

```diff
@@ -19,6 +20,7 @@
 
 __all__ = [
     'MarkerCluster',
+    'FastMarkerCluster',
```

Comment:
There's some significant differences - the main one being that the FastMarkerCluster object calls `super(FastMarkerCluster, self).__init__([])` - so no children are added and full functionality isn't available (for instance, calling `get_bounds()` returns [[None, None], [None, None]]`. I'd recommend keeping both and caveating FastMarkerCluster as being quick to draw 000's of points but loses functionality.